src/lib.rs

```diff
@@ -13,22 +13,20 @@
 #![deny(missing_docs)]
 
 extern crate cast;
-extern crate core;
 extern crate embedded_hal as hal;
 pub extern crate i2cdev;
-pub extern crate spidev;
-pub extern crate serial_unix;
-pub extern crate serial_core;
 pub extern crate nb;
-
+pub extern crate serial_core;
+pub extern crate serial_unix;
+pub extern crate spidev;
```

Comment:
Since you have upgraded to edition 2018, you can also use `pub use nb` and so on here instead of `extern crate`.